modules/machine-config-daemon-metrics.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-nodes-machine-config-daemon-metrics.adoc
+
+[id="machine-config-daemon-metrics_{context}"]
+= Machine Config Daemon metrics
+
+Beginning {product-title} 4.3, the Machine Config Daemon provides a set of metrics. These metrics can be accessed using the Prometheus Cluster Monitoring stack.
+
+The following table describes this set of metrics. Note that:
```

Comment:
I recommend a `[NOTE]` here, instead of a bulleted list.

Reply:
Thanks @sheriff-rh . I'll rather keep it like this, since those details closely pertain to the table, and are not a sort of side note.

Reply:
Changed my mind and going with your suggestion. Substituted two bullet points with two [NOTE]s.